src/test/java/LeaderboardTest/LeaderboardTest.java

```diff
@@ -24,6 +53,7 @@ public void testGetAllUsersNonEmpty() {
     }
     @Test
     public void testLeaderboardTwoUsers() {
+        Leaderboard newBoard = interactor.updateLeaderboard();
```

Comment:
is this test complete?

Reply:
Yeah everything else seems good except for the incomplete test